examples/delay-pod.js

```diff
@@ -0,0 +1,81 @@
+import { sleep } from 'k6';
+import { Kubernetes } from 'k6/x/kubernetes';
+import { PodDisruptor } from '../src/pod.js';
+import { DeploymentHelper } from '../src/helpers.js';
+import  http from 'k6/http';
+
+const app = "httpbin"
+const image = "kennethreitz/httpbin"
+
+export function setup() {
+  const k8sClient = new Kubernetes()
+
+  // create a random namespace
+  const namespace = "k6-"+Math.random().toString(32).slice(2, 7);
+  k8sClient.namespaces.create({name: namespace})
+
+  // create a test deployment
+  const helper = new DeploymentHelper(k8sClient, app, namespace, image, 2)
```

Comment:
Speaking about APIs.

I'm not sure how idiomatic it's to pass the client this way (keep in mind I'm no Javascript expert). Maybe a method called `.WithClient()` is better? Idk.

Also, it wasn't clear to me, that this helper was creating a service too (until I read the following lines getting the IP)

Reply:
Also, do you see something like this as something we wanna support? Or it's a placeholder for now?

Reply:
I'm not using an idiomatic JavaScript on purpose as this API will be re-implemented in go in `xk6-chaos`. In any case, fluent APIs are not really that common in JavaScript.

Reply:
> Also, do you see something like this as something we wanna support? Or it's a placeholder for now?

I think we should probably support this in `xk6-kubernetes`, maybe under a `helpers` package. I think it will facilitate the implementation of the setup for tests. Just look as